msal4j-brokers/src/test/java/test/ProofOfPossessionTest.java

```diff
@@ -0,0 +1,192 @@
+package test;
+
+import com.microsoft.aad.msal4j.*;
+import com.microsoft.aad.msal4jbrokers.MsalRuntimeBroker;
+import infrastructure.SeleniumExtensions;
+import labapi.LabUserProvider;
+import labapi.User;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Collections;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+@Test()
+public class ProofOfPossessionTest {
+    private final static String MICROSOFT_AUTHORITY_ORGANIZATIONS =
+            "https://login.microsoftonline.com/organizations/";
+    private final static String GRAPH_DEFAULT_SCOPE = "https://graph.windows.net/.default";
```

Comment:
Generally better to use "User.Read" scope from Graph, as it's quite harmless, as opposed to ".default" which gives access to "everything"